packages/botonic-plugin-flow-builder/src/index.ts

```diff
@@ -40,10 +43,8 @@ export default class BotonicPluginFlowBuilder implements Plugin {
   ) => Promise<void>
 
   constructor(readonly options: BotonicPluginFlowBuilderOptions) {
-    this.cmsApi = new FlowBuilderApi({
-      url: options.flowUrl,
-      flow: options.flow,
-    })
+    this.flowUrl = options.flowUrl
+    this.flow = options.flow
```

Comment:
Juat to know, in which case will you have the flow before? In fact, this class is used to fetch the flow among other things...

Reply:
Yes, you're right and I had the same doubt...! I find it also weird, but maybe this was something used in the initial versions and now is no longer needed. I'm keeping it just to ensure the logic remain the same as it was before this refactor.
Maybe @manuelalferez, @vanbasten17 or @AinaVendrell know why this was done this way.

Reply:
Is for the case that you have a local flow without flowbuilder backend only for development.

Reply:
this is a WIP project, we have it noted in order to keep improving it, both code and docs (and test, of course).

Reply:
We can review it in future PRs